Tests/SeleniumSetUp.py

```diff
@@ -1,6 +1,7 @@
 from selenium import webdriver
 from selenium.webdriver.firefox.options import Options
 from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
```

Comment:
name collision?

Reply:
Redid the name collision part by naming Options as the specific web browser Options